django/librarian/templates/librarian/components/edit_details.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n %}
 
 <div id="librarian-details" {% if swap %}hx-swap-oob="true"{% endif %}>
-  {# {% include "components/messages.html" %} #}
+  {% include "components/messages.html" %}
```

Comment:
Until the toasts appear above the lightbox overlay, it makes sense to continue showing the messages inside the librarian modal.